RoboticonColony/Assets/Code/Tile.cs

```diff
@@ -0,0 +1,72 @@
+﻿using UnityEngine;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+sealed public class Tile
+{
+    public int Cost { get; private set; }
+    public int OwnerID { get; private set; }
+
+    private Dictionary<ItemType, int> _Resources;
+    private int _ownerID;
+
+    /// <summary>
+    /// Constructor for tile
+    /// </summary> 
+    /// <param name="cost">Cost of the tile</param>
+    /// <param name="ore">Ore that the tile has</param>
+    /// <param name="power">Power that the tile has</param>
+    public Tile(int cost, int ore, int power)
+    {
+        Cost = cost;
+        _Resources = new Dictionary<ItemType, int>();
+        _Resources[ItemType.Ore] = ore;
+        _Resources[ItemType.Power] = power;
+        OwnerID = -1;
+    }
+
+    /// <summary>
+    /// Adds a roboticon to the tile
+    /// </summary>
+    /// <param name="newRoboticon">Roboticon to be added</param>
+    //public void AddRoboticon(Roboticon newRoboticon)
+    //{
+    //    _Resources[ItemType.Roboticon] = newRoboticon;
```

Comment:
This wouldn't even compile anyway. Read my other comment. _Resources is an itemtype>int dictionary, you can't, and shouldn't try to assign roboticon to it, as it makes no sense. ItemType.Roboticon is only ever used for keeping track of how many roboticons the player or market has in their inventory before they actually deploy it onto a tile.